tests/unit/test_utils_general.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from pathlib import Path
+from schematic.utils.general import (
+    check_synapse_cache_size,
+)
+
+import pytest
+from unittest.mock import MagicMock
+from unittest import mock
+
+
+@pytest.mark.parametrize(
+    "directory, file_sizes, expected_size",
+    [
+        ("", [1024], 1024),  # Default directory with 1 KB file
+        ("/custom/directory", [2048], 2048),  # Custom directory with 2 KB file
+        ("", [], 0),  # Empty directory
+        ("", [1024, 2048], 3072),  # Directory with multiple files (1 KB + 2 KB)
+    ],
+)
+def test_check_synapse_cache_size(mocker, directory, file_sizes, expected_size):
```

Comment:
There are separate tests in the `test_utils.py`, should I move them here?

Reply:
Yes please, those under `class TestGeneral:` look like good candidates to move over here